liferay-lima/liferay-portal
# LPS-133083 Remove feature flag for Image Editor in DL

https://issues.liferay.com/browse/LPS-133083

Just removing FF for Image Editor in DL.

To test:
- Go to DL
- Upload an image
- Click on the actions menu of the image
- Edit Image action should appear
